MDHelpers/MDStatics.cs

```diff
@@ -372,5 +374,45 @@ public static string HumanReadableSize(double len)
 
             return string.Format("{0:0.##} {1}", len, sizes[order]);
         }
+
+        /// <summary>
+        /// Get the data converter for the given type
+        /// </summary>
+        /// <param name="Type">The type to get the data converter for</param>
+        /// <returns>The default data converter for this type</returns>
+        public static IMDDataConverter GetConverterForType(Type Type)
+        {
+            if (!DataConverterCache.ContainsKey(Type))
+            {
+                DataConverterCache.Add(Type, IntGetConverterForType(Type));
+            }
+            return DataConverterCache[Type];
+        }
+
+        private static IMDDataConverter IntGetConverterForType(Type Type)
```

Comment:
I prefer an `_Internal` suffix for internal functions (Int is too close to the integer type), but that's something that can be fixed later along with an addition to the contribution guidelines.